Content.Server/GameTicking/GamePresets/PresetTraitor.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Content.Server.GameObjects.Components.GUI;
+using Content.Server.GameObjects.Components.Items.Storage;
+using Content.Server.GameObjects.Components.PDA;
+using Content.Server.GameTicking.GameRules;
+using Content.Server.Interfaces.Chat;
+using Content.Server.Interfaces.GameTicking;
+using Content.Server.Mobs.Roles.Traitor;
+using Content.Server.Objectives.Interfaces;
+using Content.Server.Players;
+using Content.Shared.GameObjects.Components.Inventory;
+using Content.Shared.GameObjects.Components.PDA;
+using Content.Shared.Roles;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+using Robust.Shared.Maths;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.GameTicking.GamePresets
+{
+    public class PresetTraitor : GamePreset
+    {
+        [Dependency] private IGameTicker _gameticker = default!;
+        [Dependency] private IChatManager _chatManager = default!;
+        [Dependency] private IPrototypeManager _prototypeManager = default!;
+        [Dependency] private IRobustRandom _random = default!;
+
+        public override string ModeTitle => "Traitor";
+
+        //make these cvars
+        private int MinPlayers => 2;
+        private int TraitorPerPlayers => 5;
+        private int MaxTraitors => 4;
+        private int CodewordCount => 2;
+        private int StartingTC => 20;
+
+        private string[] Codewords => new[] {"cold", "winter", "radiator", "average", "furious"};
+        private List<TraitorRole> _traitors = new ();
```

Comment:
Maybe read CodeWords from yaml? idk

Also both can be readonly

Reply:
wouldn't know where to write up codewords in yaml
also `Codewords` is a property, should i make it a field and make it readonly?